scriptForClasses.js

```diff
@@ -0,0 +1,97 @@
+const list = document.querySelector('ul');
+const form = document.querySelector('form');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+class Books {
+  constructor() {
+    this.bookList = [];
+  }
+
+  addNewBook(newBook) {
+    return this.bookList.push(newBook);
+  }
+
+  removeBooksFromList() {
+    const removeButtons = document.getElementsByClassName('remove');
+    // eslint-disable-next-line no-use-before-define
+    removeBook(removeButtons);
+    // eslint-disable-next-line no-unused-expressions
+    this;
+  }
+}
+
+const books = new Books();
+
+function addToLocalStorage(books) {
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function appendBooksToList() {
+  list.innerHTML = '';
+  books.bookList.forEach((book, index) => {
+    const li = document.createElement('li');
+
+    if (index % 2 === 1) {
+      li.classList = 'bg-white';
+    }
+
+    li.innerHTML = `
+    <p>"<span>${book.title}</span>" </p>
+    <p>by ${book.author}</p>
+    <button class="remove">Remove</button>
+    `;
+    list.appendChild(li);
+  });
+}
+
+function updateDomAndLocalStorage() {
+  appendBooksToList();
+  localStorage.clear();
+  addToLocalStorage(books);
+  books.removeBooksFromList();
+}
+
+function removeBook(removeButtons) {
+  for (let i = 0; i < removeButtons.length; i += 1) {
+    const button = removeButtons[i];
+    button.addEventListener('click', (event) => {
+      const element = event.target.parentElement.firstElementChild.firstElementChild.innerText;
+      for (let i = 0; i < books.bookList.length; i += 1) {
+        if (element === books.bookList[i].title) {
+          books.bookList.splice(i, 1);
+        }
+      }
+      updateDomAndLocalStorage();
+    });
+  }
+}
+
+/* eslint max-classes-per-file: ["error", 2] */
```

Comment:
- [ ] We were supposed to make all these functions, `class methods`, so, since you have some of these methods already in your class, kindly feel free to remove them but make sure that the functionality of your app stays the same.

![Screenshot from 2022-01-05 01-40-17](https://user-images.githubusercontent.com/56886064/148138611-f352f835-b6ea-48a0-872b-fed3cdef51fe.png)

Reply:
## Hi @descholar-ceo 
Thanks for your feedback. Could you elaborate on your request, please? 

Do you mean we should remove the methods from the class or add all the remaining functions as methods?

> * [ ]  We were supposed to make all these functions, `class methods`, so, since you have some of these methods already in your class, kindly feel free to remove them but make sure that the functionality of your app stays the same.
> 
> ![Screenshot from 2022-01-05 01-40-17](https://user-images.githubusercontent.com/56886064/148138611-f352f835-b6ea-48a0-872b-fed3cdef51fe.png)

Reply:
This issue is well resolved! ✔️